setup/ssl.sh

```diff
@@ -0,0 +1,94 @@
+ssl() {
+  if [ ! -d "nginx/conf/customs" ]; then
```

Comment:
check with $CURRENT_DIR

```suggestion
  if [ ! -d "$CURRENT_DIR/nginx/conf/customs" ]; then
```